File: PHP/DataPemain.php
<?php
//class data pemain inheritance dengan data tim
class DataPemain extends DataTim{
    //berisi nomor dan posisi, sisanya sudah inherit dari data tim
    private $nomor = 0;
    private $posisi = 0;
    
    //constructor
    public function __construct($nomor = 0, $posisi = 0){
        $this->nomor = $nomor;
        $this->posisi = $posisi;
    }
    
    //fungsi untuk menyimpan dan memanggil atribut dalam class
    public function setNomor($nomor){
        $this->nomor = $nomor;
    }

    public function getNomor(){
        return $this->nomor;
    }

    public function setPosisi($posisi){
        $this->posisi = $posisi;
    }

    public function getPosisi(){
        return $this->posisi;
    }
}

?>

File: Java/DataPemain.java
public class DataPemain extends DataTim{
    private int nomor;
    private String posisi;

    public DataPemain(){
        this.nomor = 0;
        this.posisi = "";
    }

    public void setNomor(int no){
        this.nomor = no;
    }

    public int getNomor(){
        return nomor;
    }

    public void setPosisi(String p){
        this.posisi = p;
    }

    public String getPosisi(){
        return posisi;
    }
}